helpdesk/views/public.py

```diff
@@ -58,6 +59,19 @@ def homepage(request):
         except Queue.DoesNotExist:
             queue = None
         initial_data = {}
+
+        # add pre-defined data for public ticket
+        if getattr(settings, 'HELPDESK_PUBLIC_TICKET_QUEUE', None) is not None:
+            # get the requested queue; return an error if queue not found
+            try:
+                queue = Queue.objects.get(slug=settings.HELPDESK_PUBLIC_TICKET_QUEUE)
+            except Queue.DoesNotExist:
+                return HttpResponse(status=500)
+        if getattr(settings, 'HELPDESK_PUBLIC_TICKET_PRIORITY', None) is not None:
+            initial_data['priority'] = settings.HELPDESK_PUBLIC_TICKET_PRIORITY
+        if getattr(settings, 'HELPDESK_PUBLIC_TICKET_DUE_DATE', None) is not None:
```

Comment:
What if I want the default 'due_date' to be `None`?

Reply:
Right, I missed your initial comment. Would it not just be easier to use `hasattr` as opposed to having special values?

Reply:
Err, yes, I guess so. :) I was unhappy with the special value but couldn't think of an easy way to get rid of it.

Regarding None: As we pre-set this value via a hidden in the form, I think we always end up with an empty string. I should test this though, I am not 100% sure of Django's behavior here.

Reply:
It's actually the other way round: the empty string always gets converted to `None` for datetime fields. I updated the PR accordingly.